plugins/base/base-test-utils/src/main/kotlin/renderers/SignatureUtils.kt

```diff
@@ -2,11 +2,15 @@ package signatures
 
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Element
+import utils.Tag
 import utils.TestOutputWriter
 
 fun TestOutputWriter.renderedContent(path: String = "root/example.html") =
     contents.getValue(path).let { Jsoup.parse(it) }.select("#content")
         .single()
 
 fun Element.signature() = select("div.symbol.monospace")
-fun Element.firstSignature() = signature().first()
\ No newline at end of file
+fun Element.firstSignature() = signature().first()
+
+class Parameters(vararg matchers: Any) : Tag("span", *matchers, expectedClasses = listOf("parameters"))
```

Comment:
the same

Reply:
I think it's ok since it's within `SignatureUtils` class